docs/fhir-client/0.1.0/client.js.html

```diff
@@ -187,20 +189,24 @@ <h1 class="page-title">client.js</h1>
    * @example
    *
    * // Using promises
-   * fhirClient.vread('Patient', 12345, 1).then((data) => { console.log(data); });
+   * fhirClient.vread({ resourceType: 'Patient', identifier: '12345',
+   *                    version: '1' })
+   *           .then((data) => { console.log(data); });
    *
    * // Using async
-   * let response = await fhirClient.vread('Patient', 12345, 1);
+   * let response = await fhirClient.vread({ resourceType: 'Patient',
+   *                                         identifier: '12345', version: '1' });
    * console.log(response);
    *
-   * @param {string} resource The resource type (e.g. "Patient", "Observation")
-   * @param {string} identifier The FHIR identifier for the resource
-   * @param {string} version The version identifier for the resource
+   * @param {Object} params - The request parameters.
```

Comment:
very minor, but I wonder if it should be called something else, like: "options" or "config"

(e.g.:  this method accepts a "param" called "options")

Reply:
e.g.: imagine other methods, that have both an "options" (or: "config", or whatever) Object, in addition to a dedicated, scalar param that is a String or Number.

Reply:
Oh, nvm:  ^this particular one is actually for url-params.

Reply:
Yeah, I definitely think of these as params rather than options/config.